docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.7'
+services:
+  tango-db:
+    image: tangocs/mysql:9.2.2
+    ports:
+    - "9999:3306"
+    environment:
+    - MYSQL_ROOT_PASSWORD=root
+  tango-cs:
+    image: tangocs/tango-cs
+    ports:
+    - "10000:10000"
+    environment:
+    - TANGO_HOST=localhost:10000
+    - MYSQL_HOST=tango-db:3306
+    - MYSQL_USER=tango
+    - MYSQL_PASSWORD=tango
+    - MYSQL_DATABASE=tango
+    links:
+    - "tango-db:localhost"
```

Comment:
Why do you alias `tango-db` as `localhost`? I think it may break things and it is unnecessary because you use `tango-db` in `MYSQL_HOST` anyway.